lib/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < Sequel::Model(:userdetails)
+  User.unrestrict_primary_key
```

Comment:
Why, btw?

Reply:
The primary key for userdetails is username.
Without this setting, we won't be able to set it explicitly.

https://sequel.jeremyevans.net/rdoc/classes/Sequel/Model/ClassMethods.html#method-i-unrestrict_primary_key